vega_sim/null_service.py

```diff
@@ -1006,9 +1006,6 @@ def start(self, block_on_startup: bool = True) -> None:
             time.sleep(6)
             self.process_pids = parent_conn.recv()
 
-        # Initialise the data-cache
-        self.data_cache
-
```

Comment:
?

Reply:
Just moved too after the early exit if replaying.

Couldn't think of a reason we would need it if replaying?